Common/Common.csproj

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">

```

Comment:
I've just updated `format-fixer.py` to trim the trailing newline in csproj files and also ensure the ToolsVersion isn't updated by your editor. Please run that script and then commit the changes it generates. It'll also fix the indention problem in the .cs file.